Caprica/common/CapricaStats.h

```diff
@@ -10,8 +10,8 @@ struct CapricaStats final {
   {
     size_t val;
 
-    NopIncStruct& operator++(int) { return *this; }

```

Comment:
The purpose of this is so that it's a no-op. To enable stats, just uncomment either line 18 or 19.